gen-ai/inference/stable-diffusion-rayserve-gpu/s3-pv-model-storage.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: s3-pv-model
+spec:
+  capacity:
+    storage: 1200Gi # ignored, required
+  accessModes:
+    - ReadWriteMany # supported options: ReadWriteMany / ReadOnlyMany
+  mountOptions:
+    - allow-overwrite
+    - allow-delete
+    - region us-west-2
+    - prefix stable-diffusion-2/
+  csi:
+    driver: s3.csi.aws.com # required
+    volumeHandle: s3-csi-driver-volume
+    volumeAttributes:
+      bucketName: ml-model-storage-091550601287 # Replace bucketName with the actual S3 bucket name
```

Comment:
replace this name with `<ENTER_S3_BUCKET_NAME>`
also, add a comment saying that use the S3 bucket name created by this blueprint. Verify the terraform outputs